validator/sawtooth_validator/gossip/gossip_handlers.py

```diff
@@ -61,8 +61,8 @@ def __init__(self, gossip):
     def handle(self, connection_id, message_content):
         response = GetPeersResponse()
         response.ParseFromString(message_content)
-        LOGGER.debug("got peers response message "
-                     "from %s. sending ack", connection_id)
+
+        LOGGER.debug("Got peers response message from %s", connection_id)
```

Comment:
This should be combined with the line below.  Few lines, more information.

Reply:
second